hub-templates/base-hub/values.yaml

```diff
@@ -193,3 +193,18 @@ jupyterhub:
             return super().start(*args, **kwargs)
 
         c.JupyterHub.spawner_class = CustomSpawner
+      06-docs-nginx-service: |
+          # Adds JupyterHub service to make the docs available at
+          # {HUB_URL}/services/docs
+          import os
+          host = os.environ["DOCS_SERVICE_SERVICE_HOST"]
```

Comment:
When there's a service named `docs-service`, kubernetes adds a DNS entry for `docs-service`. So you can just access `docs-service` from the hub, and that works well! No need to use environment variables.

Reply:
https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/ has some more info